src/tools/simeng/main.cc

```diff
@@ -121,19 +122,19 @@ int main(int argc, char** argv) {
     // at the start of each branch. With an instruction latency of 2 or greater,
     // the `orr` at the start of the next loop should issue/execute while the
     // preceding branch is waiting on the result from the `subs`.
-    uint32_t hex[] = {
-        // 0x321E03E0,  // orr w0, wzr, #4
-        // 0x321603E0,  // orr w0, wzr, #1024
-        0x320C03E0,  // orr w0, wzr, #1048576
-        0x320003E1,  // orr w0, wzr, #1
-        0x71000400,  // subs w0, w0, #1
-        // 0x00000000,  // invalid
-        0x54FFFFC1,  // b.ne -8
-                     // .exit:
-        0xD2800000,  // mov x0, #0
-        0xD2800BC8,  // mov x8, #94
-        0xD4000001,  // svc #0
-    };
+//    uint32_t hex[] = {
+//        // 0x321E03E0,  // orr w0, wzr, #4
+//        // 0x321603E0,  // orr w0, wzr, #1024
+//        0x320C03E0,  // orr w0, wzr, #1048576
+//        0x320003E1,  // orr w0, wzr, #1
+//        0x71000400,  // subs w0, w0, #1
+//        // 0x00000000,  // invalid
+//        0x54FFFFC1,  // b.ne -8
+//                     // .exit:
+//        0xD2800000,  // mov x0, #0
+//        0xD2800BC8,  // mov x8, #94
+//        0xD4000001,  // svc #0
+//    };
```

Comment:
Is commenting this out going to effect running aarch64 codes?

Reply:
This specific code was already commented out. The "Simple loop" at the bottom is used in one of the CI test pipelines I think which will therefore fail. If a program is provided this will not effect anything